src/Container/ServerRequestSwooleFactory.php

```diff
@@ -21,7 +28,35 @@ class ServerRequestSwooleFactory
     public function __invoke(ContainerInterface $container) : callable
     {
         return function (swoole_http_request $request) {
-            return ServerRequestFactory::fromSwoole($request);
+            // Aggregate values from Swoole request object
+            $get     = $request->get ?? [];
+            $post    = $request->post ?? [];
+            $cookie  = $request->cookie ?? [];
+            $files   = $request->files ?? [];
+            $server  = $request->server ?? [];
```

Comment:
We don't need anymore to convert the server key values in uppercase?

Reply:
The previous version wasn't converting them to uppercase, from what I could see. We can make that explicit, though, using `array_change_key_case($server, CASE_UPPER)`, though. I'm not sure what casing Swoole uses...

Reply:
I've updated to make this explicit casting in the `$server` array.